src/index.html

```diff
@@ -1,15 +1,116 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post" autocomplete="off"
+          onsubmit="onSubmit()">
```

Comment:
Теги у яких три чи більше атрибути форматуємо наступним чином. Так їх простіше читати
```suggestion
    <form 
      class="form" 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post" 
      autocomplete="off"
      onsubmit="onSubmit()"
    >
```

Reply:
fixed

Reply:
done